fern/api-reference/megaeth/megaeth-api-quickstart.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: MegaETH API Quickstart
+description: How to get started building on MegaETH and using the JSON-RPC API
+subtitle: How to get started building on MegaETH and using the JSON-RPC API
+slug: reference/megaeth-api-quickstart
+---
+
+*To use the MegaETH API you'll need to [create a free Alchemy account](https://dashboard.alchemy.com/signup) first!*
+
+## Introduction
+
+MegaETH is a high-performance Ethereum Layer 2 blockchain designed to achieve real-time transaction processing with sub-millisecond latency and extremely high throughput.
+
+## What is the MegaETH API?
+
+The MegaETH API allows interaction with the MegaETH network through a set of JSON-RPC methods. Its design is familiar to developers who have worked with Ethereum's JSON-RPC APIs, making it intuitive and straightforward to use.
+
+## Getting Started Instructions
+
+### 1. Choose a Package Manager (npm or yarn)
+
+Select a package manager to manage your project's dependencies. Choose between `npm` and `yarn` based on your preference or project requirements.
+
+<CodeGroup>
+  ```shell npm
+  # Begin with npm by following the npm documentation
+  # https://docs.npmjs.com/downloading-and-installing-node-js-and-npm
+  ```
+
+  ```shell yarn
+  # For yarn, refer to yarn's installation guide
+  # https://classic.yarnpkg.com/lang/en/docs/install
+  ```
+</CodeGroup>
```

Comment:
nit: I like how we did it for tron better: https://alchemy-preview-a5cacbba-b83e-482e-9cd0-097ccc6b2a6f.docs.buildwithfern.com/docs/reference/tron-api-quickstart#1-choose-a-package-manager-npm-or-yarn

The way its done here just makes you have to highlight, copy paste. I'm sure this is a template though, so we can update this later.